pkg/store/verifiable/store.go

```diff
@@ -229,6 +231,62 @@ func (s *StoreImplementation) GetPresentations() ([]*Record, error) {
 	return s.getAllRecords(presentationNameDataKey(""), getPresentationName)
 }
 
+// RemoveCredential removes the verifiable credential and its records containing given name.
+func (s *StoreImplementation) RemoveCredential(name string) error {
+	if name == "" {
+		return errors.New("credential name is mandatory")
+	}
+
+	id, err := s.GetCredentialIDByName(name)
+	if err != nil && !errors.Is(err, storage.ErrDataNotFound) {
+		return fmt.Errorf("get credential id using name : %w", err)
+	}
```

Comment:
why do we need this condition  `!errors.Is(err, storage.ErrDataNotFound)` ?

Reply:
Correct. we don't need this check here and it is ok to rerun this error message for missing ID too.
Fixed.